x/subscription/types/cluster.go

```diff
@@ -23,6 +23,8 @@ import "strconv"
 
 const FREE_PLAN = "free" // gets its own const because it's treated differently
 
+var SUB_USAGES = []int{0, 6, 12} // sub usages that are treated differemtly when constructing a cluster key
```

Comment:
make sure to use that also in the GetSubUsageCriterion

Reply:
done. see https://github.com/lavanet/lava/commit/a1547529b3cefd06a28a6dd754a9f3970c40496b